website/content/docs/datasources/amazon/index.mdx

```diff
@@ -0,0 +1,24 @@
+---
+description: |
+  Packer is able to fetch data from AWS. To achieve this, Packer comes with
+  data sources to retrieve AMI and secrets information.
+page_title: Amazon - Data Sources
+sidebar_title: Amazon
+---
+
+# Amazon Data Sources
+
+Packer is able to fetch data from AWS. To achieve this, Packer comes with data sources to retrieve AMI and secrets information.
+Packer supports the following data sources at the moment:
+
+- [amazon-ami](/docs/datasources/amazon/ami) - Filter and fetch an Amazon AMI to output all the AMI information.
+
+- [amazon-secretsmanager](/docs/datasources/amazon/secretsmanager) - Retrieve information
+about a Secrets Manager secret version, including its secret value.
+
+
+## Authentication
+
+The Amazon Data Sources authentication works just like for the [Amazon Builders](/docs/builders/amazon). Both
```

Comment:
How would a user only interested in using the datasource, without an amazon builder, specify the authentication for AWS? I'm thinking in the case of AWS authentication with the use of assume_role

Reply:
I'll add an example as talked in our 1:1 as well as the note about separate authentication between builders and data sources.

Reply:
The authentication for data source doesn't depend on the builder one